### PR TITLE
first version of release note 7.5.0

### PR DESCRIPTION
**Pull request by philippelaumay**
first version to be ready for BETA program, 
may be updated and refined for GA

### CODE REVIEW

File: md/release-notes.md [hunk 1]
```diff
@@ -18,21 +66,20 @@ Bonita BPM 7.5 includes a set of user-based product enhancements, bug fixes and
...
 
 
-### Full Java 8 support
-Bonita BPM 7.5.0+ is now fully Java 8 compliant.  
-Java 8 syntax and specific APIs can now be fully used in all Bonita BPM [extensions points](software-extensibility.md#stable_extension_points)  
-As a consequence, JRE or JDK 8+ is required to run. Bonita BPM 7.5.0+ cannot be operated on Java 1.7 anymore.
-
-### Removed support for Oracle Weblogic application server
+### Removed support 
+* Remove suport for Oracle Weblogic application server
```
Comment (educhastenier) [inline md/release-notes.md]: sup**p**ort

File: md/release-notes.md [hunk 2]
```diff
@@ -18,21 +66,20 @@ Bonita BPM 7.5 includes a set of user-based product enhancements, bug fixes and
...
+* httpcore is updated to httpcore 4.4.4
 
+### API updates
 
-## API updates
-
-### API removals
-Engine APIs have been cleaned up from some deprecated methods:
-* ProcessAPI.addComment() - replacement method is [ProcessAPI.addProcessComment()](http://documentation.bonitasoft.com/javadoc/api/${varVersion}/org/bonitasoft/engine/api/ProcessRuntimeAPI.html#addProcessComment-long-java.lang.String-)
+* API removals Engine APIs have been cleaned up from some deprecated methods:
```
Comment (educhastenier) [inline md/release-notes.md]: Why did you merge the title with the first sentence?

File: md/release-notes.md [hunk 3]
```diff
@@ -1,14 +1,62 @@
...
 ## Technology updates
 
-- httpclient is updated to httpclient 4.5.2
-- httpcore is updated to httpcore 4.4.4
+### Full Java 8 support
+Bonita BPM is fully compatible with java 8 in production and development environment. It allows to write java 8 extension points (Connectors, Actor Filters, Custom Pages, ...). Java 8 syntax and specific APIs can now be fully used in all Bonita BPM [extensions points](software-extensibility.md#stable_extension_points)  
+As a consequence, JRE or JDK 8+ is required to run. Bonita BPM 7.5.0+ cannot be operated on Java 1.7 anymore.
+
+### Support of Microsoft Edge
+Bonita web tools and user interfaces are fully compatible with the Microsoft Edge and IE 11 browsers (UI Designer editor, All artefacts generated by UID (Forms, Pages, ...), Bonita Portal).
```
Comment (educhastenier) [inline md/release-notes.md]: ~~A~~**a**ll

File: md/release-notes.md [hunk 4]
```diff
@@ -1,14 +1,62 @@
...
 
-Bonita BPM 7.5 includes a set of user-based product enhancements, bug fixes and important technology updates.
+Bonita BPM 7.5 includes a set of new features related to the Living Application development and deployment and UI personalization in addition to important technology updates and bug fixes.
 
+<a id="living-application-development-and-deployment"/>
+
+## Living Application Development and Deployment
+
+### Integration of SSO SAML 2.0
+Bonita BPM works with Single-Sign-On (SSO) solutions using SAML V2 protocol `(Teamwork edition)`
```
Comment (educhastenier) [inline md/release-notes.md]: `(Teamwork, Efficiency and Performance editions)`

**Comment by educhastenier** [comment]
retest this please

File: md/release-notes.md [hunk 5]
```diff
@@ -1,14 +1,62 @@
...
+* Being able to use standard authentication service 
+* Can connect to Bonita apps with my company credentials (per tenant configuration)
+* Common SAML 2.0 Identity providers : ADFS (MS Active Directory Federation Services), SalesForce, Open AM, Google Auth,...
+
+### New wizard to import a process in the studio & compare the artefacts 
+Provide a wizard when importing a .bos that display the duplicate artefacts (forms, connectors, ...)  and ask for actions to the users. Usual actions are proposed to user for every artefacts : keep existing or overwrite with the new import.
+
+Capabilities :
+* It eases collaboration between teams who works with .bos exchange
+* It eases the collaboration by file exchange and reduce drastically the risk of overwrite error during import of a .bos.
```
Comment (educhastenier) [inline md/release-notes.md]: reduce**s**

File: md/release-notes.md [hunk 6]
```diff
@@ -1,14 +1,62 @@
...
+
+### Industrialization of Living Application (in Studio)
+In Bonita BPM Studio, provide an editor to create the Living Application navigation without having to go on the Portal administration. This is the first step towards the full Application artefacts development into the Bonita BPM Studio in future versions `(Efficiency edition)`
```
Comment (educhastenier) [inline md/release-notes.md]: `(Efficiency and Performance editions)`

File: md/release-notes.md [hunk 7]
```diff
@@ -1,14 +1,62 @@
...
+
+Capabilities : 
+* Create and edit LA navigation descriptor in Studio. It is the first step to industrialization of LA.
```
Comment (educhastenier) [inline md/release-notes.md]: Define LA before using it

File: md/release-notes.md [hunk 8]
```diff
@@ -1,14 +1,62 @@
...
+* can store it in your VCS
+* can (ultimately) industrialize the continuous integration of your application
+* can preview and push to deploy on the local Portal to see the Application immediately. 
+
+<a id="ui-personalization"/> 
+
+## UI personalization
+
+### New management of date/time
+New management of date and time to properly manage application used on multiple timezone. It now follows the philosophy of Java 8 date management.
```
Comment (educhastenier) [inline md/release-notes.md]: application**s**

**Comment by educhastenier** [comment]
General remark: sentence formulation does not seem to be consistent with the way we usually turn phrases... check with other pages

File: md/release-notes.md (hunk 1 above)
Comment (philippelaumay) [reply md/release-notes.md]: ok

File: md/release-notes.md (hunk 7 above)
Comment (laurentleseigneur) [reply md/release-notes.md]: keep Living Application instead of using LA

File: md/release-notes.md [hunk 9]
```diff
@@ -1,14 +1,62 @@
...
+Capabilities :
+* It eases collaboration between teams who works with .bos exchange
+* It eases the collaboration by file exchange and reduce drastically the risk of overwrite error during import of a .bos.
+
+### Industrialization of Living Application (in Studio)
+In Bonita BPM Studio, provide an editor to create the Living Application navigation without having to go on the Portal administration. This is the first step towards the full Application artefacts development into the Bonita BPM Studio in future versions `(Efficiency edition)`
+
+Capabilities : 
+* Create and edit LA navigation descriptor in Studio. It is the first step to industrialization of LA.
+* can construct application navigation from inside Studio based on existing artefacts.
```
Comment (laurentleseigneur) [inline md/release-notes.md]: > existing artefacts

artefacts -> artifacts
explain also that existing means inside current workspace and not deployed on portal

File: md/release-notes.md (hunk 8 above)
Comment (laurentleseigneur) [reply md/release-notes.md]: ... used across ... ?

File: md/release-notes.md [hunk 10]
```diff
@@ -1,14 +1,62 @@
...
+
+<a id="ui-personalization"/> 
+
+## UI personalization
+
+### New management of date/time
+New management of date and time to properly manage application used on multiple timezone. It now follows the philosophy of Java 8 date management.
+
+Capabilities :
+* Benefit from the Java 8 improvement of date/time management in order to manager better timezones in UI (forms and pages). Ensure the use cases of :
```
Comment (laurentleseigneur) [inline md/release-notes.md]: to better manage

File: md/release-notes.md [hunk 11]
```diff
@@ -1,14 +1,62 @@
...
+    * define and display a flight arrival date (local timezone)
+* Note this update does not change the way dates are displayed on the Bonita Portal.
 
 <a id="technology-updates"/>
 
 ## Technology updates
 
-- httpclient is updated to httpclient 4.5.2
-- httpcore is updated to httpcore 4.4.4
+### Full Java 8 support
```
Comment (laurentleseigneur) [inline md/release-notes.md]: in fact Java8 is now required

File: md/release-notes.md (hunk 3 above)
Comment (laurentleseigneur) [reply md/release-notes.md]: artifacts

File: md/release-notes.md [hunk 12]
```diff
@@ -18,21 +66,20 @@ Bonita BPM 7.5 includes a set of user-based product enhancements, bug fixes and
...
 Refer to [full software requirements](hardware-and-software-requirements.md) for more details on supported environments.
+* Remove support for Intyernet Explorer 9 and 10
+Bonita BPM 7.5.0+ does not support Internet Explorer 9 and 10 anymore.  
+Refer to [full software requirements](hardware-and-software-requirements.md) for more details on supported environments.
 
-### Other updates
+### Other dependency updates
 * Hibernate has been updated from version 4.2.13 to 4.3.11, to support JPA 2.1 specification
+* httpclient is updated to httpclient 4.5.2
+* httpcore is updated to httpcore 4.4.4
```
Comment (laurentleseigneur) [inline md/release-notes.md]: check that Spring has also been updated

**Review by laurentleseigneur — CHANGES_REQUESTED** [review]

File: md/release-notes.md (hunk 12 above)
Comment (cpuy) [reply md/release-notes.md]: Spring 4.3.7.RELEASE

File: md/release-notes.md [hunk 13]
```diff
@@ -1,38 +1,78 @@
...
 
-### Other updates
-* Hibernate has been updated from version 4.2.13 to 4.3.11, to support JPA 2.1 specification
+Only in the case you have created custom widgets using this silent library, you need to update them by adding UI Bootstrap js as an asset.
 
+### Removed support 
+* Remove support for Oracle Weblogic application server
+Bonita BPM 7.5.0+ does not support Oracle Weblogic application server anymore.  
+Refer to [full software requirements](hardware-and-software-requirements.md) for more details on supported environments.
+* Remove support for Intyernet Explorer 9 and 10
```
Comment (cpuy) [inline md/release-notes.md]: typo `Intyernet`

Comment (laurentleseigneur) [reply md/release-notes.md]: Intyernet

**Review by laurentleseigneur — CHANGES_REQUESTED** [review]

**Pull request merged**